app/models/page.rb

```diff
@@ -39,5 +40,13 @@ def plugins
       plugin_class.where(page_id: id).to_a
     end.flatten
   end
+
+  private
+
+  def switch_layouts
+    if changed.include?("liquid_layout_id") || changed.include?("liquid_layout")
```

Comment:
yeah that was superstition when I was trying to track down a bug. taking it out now.